### PR TITLE
Add direct link to message onBotMention

### PR DESCRIPTION
**Pull request by sporkwitch**
Implement feature request #318
Add a blank line and then direct link to the individual message in which
the bot was mentioned.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -166,9 +166,9 @@ bot.on('messageCreate', async msg => {
   const staffMention = (config.pingOnBotMention ? utils.getInboxMention() : '');
 
   if (mainGuilds.length === 1) {
-    content = `${staffMention}Bot mentioned in ${msg.channel.mention} by **${msg.author.username}#${msg.author.discriminator}**: "${msg.cleanContent}"`;
+    content = `${staffMention}Bot mentioned in ${msg.channel.mention} by **${msg.author.username}#${msg.author.discriminator}**: "${msg.cleanContent}"\n\nhttps:\/\/discordapp.com\/channels\/${msg.channel.guild.id}\/$msg.channel.id}\/${msg.id}`;
   } else {
-    content = `${staffMention}Bot mentioned in ${msg.channel.mention} (${msg.channel.guild.name}) by **${msg.author.username}#${msg.author.discriminator}**: "${msg.cleanContent}"`;
+    content = `${staffMention}Bot mentioned in ${msg.channel.mention} (${msg.channel.guild.name}) by **${msg.author.username}#${msg.author.discriminator}**: "${msg.cleanContent}"https:\/\/discordapp.com\/channels\/${msg.channel.guild.id}\/$msg.channel.id}\/${msg.id}`;
```
Comment (reboxer) [inline src/main.js]: Shouldn't this line have two newlines for the jump link too?

**Comment by sporkwitch** [comment]
Good catch, I'll fix it when I finish this cigarette and get back inside. 
v / r
Robert Klebes
Mobile: 571-492-8639
https://keybase.io/sporkwitch
https://www.linkedin.com/in/klebes

File: src/main.js (hunk 1 above)
Comment (sporkwitch) [reply src/main.js]: Fixed

Comment (sporkwitch) [reply src/main.js]: Fixed

**Pull request merged**